Vehicle/Sensor_Acquisition_Board_Rear_rev5/platformio.ini

```diff
@@ -0,0 +1,25 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:teensy40]
+platform = teensy
+board = teensy40
+framework = arduino
+build_unflags = -std=gnu++11
+build_flags = -std=c++17
+
+lib_deps =
+    SPI
+    https://github.com/hytech-racing/shared_firmware_interfaces.git#feature/vn-interface-class
```

Comment:
Should this be merged on the shared_firmware_interfaces repo to minimize branch dependencies in platformio.ini?